### PR TITLE
Import injected batch data from file

### LINKED ISSUE #1300: Flexible Chain Initialization Handling
## Context
Today, a batch, called "initial batch", of transactions is sequenced in L1 before an L2 is created. The batch contains transactions such as bridge initialization. For rollups and validiums, the initial batch is read from L1 endpoint and written to the database in [`HandleInitialSequenceBatches`](https://github.com/0xPolygonHermez/cdk-erigon/blob/fe753f28b291f0d2e1e2221e4ac8e061d70f87ff/zk/stages/stage_l1_sequencer_sync.go#L248-L286), and [injected](https://github.com/0xPolygonHermez/cdk-erigon/blob/fe753f28b291f0d2e1e2221e4ac8e061d70f87ff/zk/stages/stage_sequence_execute_injected_batch.go#L25-L84) in execution stage. 

## New feature
In some scenarios, e.g. sovereign chains, the initialization transaction are not posted to L1. Instead, they could be written to a file or directly baked in the genesis. `cdk-erigon` should be able to handle both cases. 

#### Batch injection
The transactions should be read from a local file specified in configuration, and executed in execution stage. This logic could be added to [`processInjectedInitialBatch`](https://github.com/0xPolygonHermez/cdk-erigon/blob/fe753f28b291f0d2e1e2221e4ac8e061d70f87ff/zk/stages/stage_sequence_execute_injected_batch.go#L25-L84).
Notice that we can allow the file to be empty. When the file is provided but empty, it implies the initialization txns are already baked in genesis, and the same code logic will be able to handle both cases automatically.

#### Forkid 
Forkid is used in many places in cdk-erigon and it is initialized in injected batch. For some chains, forkid isn't even required. However, without picking a forkid, cdk-erigon might freeze or break at some stage (we should fix this in the end). For now, a quick work around is to pick the latest forkid and [write it to db](https://github.com/0xPolygonHermez/cdk-erigon/blob/fe753f28b291f0d2e1e2221e4ac8e061d70f87ff/zk/hermez_db/db.go#L1048-L1050) when forkid isn't required, e.g. sovereign chains.

### PR DESCRIPTION
**Pull request by Stefan-Ethernal**
This PR introduces the `zkevm.initial-batch.config` flag. It expects the `firstBatchData` provided, which acts like `L1InjectedBatch` for the pessimistic proofs consensus.

Example of `firstBatchData`, that can be found in the `create_rollup_output.json` when deploying the zkevm-contracts:

```json
{
  "batchL2Data": "0xf9010680808401c9c38094adc8abe406e5bef98e78083e5dea13f3014e39f980b8e4f811bff7000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a40d5f56745a118d0906a34e69aec8c0db1cb8fa000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c0000000000000000000000000000000000000000000000000000000000000000080808000000000000000000000000000000000000000000000000000000005ca1ab1e0000000000000000000000000000000000000000000000000000000005ca1ab1e13ff",
  "globalExitRoot": "0xad3228b676f7d3cd4284a5443f17f1962b36e491b30a40b2405849e597ba5fb5",
  "timestamp": 1729085887,
  "sequencer": "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266",
  "l1BlockNumber": 83,
  "l1BlockHash": "0x8e668d4f32f8786835c5a27f6131c29560fad5d14746f723b649c959acc04eb8",
  "l1ParentHash": "0xe02a6e50f1fb144b861c020b4ccde235744a0179a3e9c90d57026d94f3a4b1ba"
 }
```

Closes #1300.

## Notes

- zkevm-contracts - https://github.com/0xPolygonHermez/zkevm-contracts/tree/feature/sovereign-bridge
- Kurtosis CDK  - https://github.com/0xPolygon/kurtosis-cdk/tree/jhilliard/pp-integration

### CODE REVIEW

File: cmd/utils/flags.go [hunk 1]
```diff
@@ -733,6 +733,11 @@ var (
 		Usage: "The multiplier to reduce the SMT depth by when calculating virtual counters",
 		Value: 0.6,
 	}
+	PessimisticProofsCfgFile = cli.StringFlag{
+		Name:  "zkevm.pessimistic-proofs.config",
```
Comment (cffls) [inline cmd/utils/flags.go]: Technically, a chain with full execution proof can also use this file, e.g. a full execution proof chain whose initial batch is baked in genesis, and this file will be empty. A name like `zkevm.initial-batch.config` sounds a more generic name that can be used by both pessimistic and full execution proof?

File: zk/types/zk_types_test.go [hunk 2]
```diff
@@ -50,3 +52,35 @@ func Test_L1InjectedBatchMarshallUnmarshall(t *testing.T) {
 
 	require.Equal(t, input, result)
 }
+
+func Test_L1InjectedBatch_UnmarshalJSON(t *testing.T) {
```
Comment (cffls) [inline zk/types/zk_types_test.go:58]: Thanks for adding this test. If you could add a test for empty file that would be great.

**Review by cffls — COMMENTED** [review]
LGTM overall.

File: zk/types/zk_types_test.go (hunk 2 above)
Comment (Stefan-Ethernal) [reply zk/types/zk_types_test.go:58]: Fixed https://github.com/0xPolygonHermez/cdk-erigon/pull/1312/commits/13865ecf710b2539273a9356c051c6e27d7cf77f

File: cmd/utils/flags.go (hunk 1 above)
Comment (Stefan-Ethernal) [reply cmd/utils/flags.go]: Fixed https://github.com/0xPolygonHermez/cdk-erigon/pull/1312/commits/13865ecf710b2539273a9356c051c6e27d7cf77f

**Review by cffls — DISMISSED** [review]
LGTM, thanks!

**Review by cffls — APPROVED** [review]
